src/ssram.c

```diff
@@ -148,53 +164,96 @@ void ice_ssram_deinit(void) {
     spi_deinit(SPI_SSRAM);
 }
 
-/// Enqueue a SPI write operation performed asynchronously.
-/// Call ice_ssram_wait() after this to perform a blocking write until completion.
-void ice_ssram_write(int cs_pin, uint32_t dest_addr, const void* src, uint32_t size) {
+void ice_ssram_output_command(int cs_pin, const uint8_t* command, uint32_t command_size, const void* data, uint32_t data_size, bool async) {
```

Comment:
indeed, this spi write function name was clashing with the write operation toward SRAM.